docs/configuration.md

```diff
@@ -1,39 +1,43 @@
 # Configuration.
 
-All customisation is done via a YAML configuration file at `.beetbox/config.yml`. If this file doesn't exist it is automatically created the first time you start up the VM.
-Configuration is applied hierarchically so any attribute set in your project config file will override a lower level default setting.
+All customisation is done via a YAML configuration file at `.beetbox/config.yml`. If this file doesn't exist, it is automatically created the first time you start up the VM.
 
-Each ansible role beetbox uses has a set of defaults eg. the PHP role -- https://github.com/geerlingguy/ansible-role-php/blob/master/defaults/main.yml
-so any of these YAML attributes could be imported into your project `config.yml` to override the default setting.
+Configuration is applied hierarchically so any attribute set in your project config file will override a lower level default setting.
 
-eg. `php_date_timezone: "Antarctica/South_Pole"`
+## Configuration from used roles
+Each ansible role beetbox uses has a set of defaults (eg. the PHP role  https://github.com/geerlingguy/ansible-role-php/blob/master/defaults/main.yml)
+so any of these YAML attributes could be imported into your project `config.yml` to override the default setting:
+```
+php_date_timezone: "Antarctica/South_Pole"
+```
 
-The full list ansible roles beetbox implements is here -- https://github.com/drupalmel/beetbox/blob/master/ansible/beetbox.config.yml#L29
+The full list of ansible roles beetbox implements is here: [https://github.com/drupalmel/beetbox/blob/master/ansible/beetbox.config.yml#L29](https://github.com/drupalmel/beetbox/blob/master/ansible/beetbox.config.yml#L29)
```

Comment:
This link needs to be updated to new location --> https://github.com/drupalmel/beetbox/blob/master/provisioning/ansible/beetbox.config.yml